web/themes/custom/pineapple/sass/components/_card.scss

```diff
@@ -4,6 +4,16 @@
   height: 100%;
   overflow: hidden;
 
+  &:hover,
+  &:focus {
+    cursor: pointer;
+    background-color: darken($light-gray, 5%);
+
+    .card-image-container {
+      opacity: 90%;
```

Comment:
Although 90% equals 0.9, i think you should use 0.9 because this is standard implementation: https://www.w3schools.com/CSSref/css3_pr_opacity.asp some browsers like IE or safari might have issues with having a % value as opposed to a decimal number